includes/discovery/sensors/dbm/infinera-groove.inc.php

```diff
@@ -29,4 +29,10 @@
         $value = $data['portRxOpticalPower'];
         discover_sensor(null, 'dbm', $device, $oid, 'portRxOpticalPower.' . $index, 'infinera-groove', $descr, null, '1', null, null, null, null, $value);
     }
+    if (is_numeric($data['portTxOpticalPower']) && $data['portTxOpticalPower'] != -99) {
```

Comment:
Looks like Receive Power is available in the MIB as well, can you add that in?

Reply:
> Looks like Receive Power is available in the MIB as well, can you add that in?

Receive power is already there my friend. I am simply adding the transmit power.